.github/workflows/release.yml

```diff
@@ -16,134 +16,206 @@ on:
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
         with:
-          fetch-depth: 0
-      - name: Install dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y autoconf automake build-essential cmake gcc-11 gettext gperf libtool-bin locales ninja-build pkg-config unzip
-      - if: github.event_name == 'push' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name != 'nightly')
+          image: tonistiigi/binfmt:latest
+          platforms: arm64 # Platforms to install (arm64,riscv64,arm ; default all)
+      -
+        name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true # build -> buildx automatically
+      -
+        if: github.event_name == 'push' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name != 'nightly')
         run: printf 'NVIM_BUILD_TYPE=Release\n' >> $GITHUB_ENV
-      - if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name == 'nightly')
+      -
+        if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name == 'nightly')
         run: printf 'NVIM_BUILD_TYPE=RelWithDebInfo\n' >> $GITHUB_ENV
-      - name: Build release
-        id: build
+      -
+        name: Build Neovim and Create .tar.gz for ${{ matrix.arch }}
+        run: |
+          docker build --platform linux/${{ matrix.arch }} \
+                 --build-arg CC=gcc-11 \
+                 --build-arg CMAKE_BUILD_TYPE=${NVIM_BUILD_TYPE} \
+                 --build-arg CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX:PATH=" \
+                 --build-arg ARCH=${{ matrix.arch }} \
+                 -f .github/workflows/dockerfile-output-targz \
+                 -o $GITHUB_WORKSPACE/build .
+      -
+        name: Output version and release for GH release notes
+        id: metadata
         run: |
-          CC=gcc-11 make CMAKE_BUILD_TYPE=${NVIM_BUILD_TYPE} CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX:PATH="
-          printf '::set-output name=version::%s\n' "$(./build/bin/nvim --version | head -n 3 | sed -z 's/\n/%0A/g')"
-          printf '::set-output name=release::%s\n' "$(./build/bin/nvim --version | head -n 1)"
-          make DESTDIR="$GITHUB_WORKSPACE/build/release/nvim-linux64" install
-          cd "$GITHUB_WORKSPACE/build/release"
-          tar cfz nvim-linux64.tar.gz nvim-linux64
-      - uses: actions/upload-artifact@v2
+          cd $GITHUB_WORKSPACE/build/
+          tar xzf nvim-linux-${{ matrix.arch }}.tar.gz
+          printf '::set-output name=version::%s\n' "$(./nvim-linux-${{ matrix.arch }}/bin/nvim --version | head -n 3 | sed -z 's/\n/%0A/g')"
+          printf '::set-output name=release::%s\n' "$(./nvim-linux-${{ matrix.arch }}/bin/nvim --version | head -n 1)"
```

Comment:
Would enjoy an if arch == amd64 since we can't actually run the arm bin and only need to run something once. Currently it "harmlessly errors" for arm.

Reply:
Fixed